app/views/layouts/tpi/_header.html.erb

```diff
@@ -7,23 +7,25 @@
     },
     {
       entry: 'Publications and news',
-      path: '/tpi/publications',
+      path: url_for(controller: 'publications', only_path: true)
     },
     {
       entry: 'About',
       content: @about_pages,
+      path: url_for(controller: 'home', action: 'about', only_path: true)
```

Comment:
why not use `tpi_about_path`? the same for `tpi_publications_path`, `tpi_search_path`